hybrisprovider.cpp

```diff
@@ -1039,6 +1045,8 @@ void HybrisProvider::xtraDownloadFinished()
         QByteArray xtraData = m_xtraDownloadReply->readAll();
         m_xtra->inject_xtra_data(xtraData.data(), xtraData.length());
 
+        qCDebug(lcGeoclueHybris) << "xtra download injected";
```

Comment:
Data is probably too much to write here, but maybe we should print the lenght of the injected data?, "... injected with lenght X"?